### PR TITLE
added Safari Light, Safari Dark themes

### PR DESCRIPTION
**Pull request by vladstudio**
as discussed here:
https://github.com/mikekazakov/nimble-commander/discussions/107

### CODE REVIEW

File: Source/NimbleCommander/NimbleCommander/Resources/Config.json [hunk 1]
```diff
@@ -866,7 +866,205 @@
...
+                        "filePanelsListRegularEvenRowBackgroundColor": "#FFFFFF",
+                        "filePanelsListRegularOddRowBackgroundColor": "#F6F6F6",
+                        "filePanelsBriefFont": "@systemFont, 13",
+                        "filePanelsBriefGridColor": "#E6E6E6",
+                        "filePanelsBriefRegularEvenRowBackgroundColor": "#FFFFFF",
+                        "filePanelsBriefRegularOddRowBackgroundColor": "#F6F6F6",
+                        "filePanelsBriefFocusedActiveItemBackgroundColor": "#0064E1",
+                        "filePanelsBriefFocusedInactiveItemBackgroundColor": "#0064E155",
+                        "filePanelsBriefSelectedItemBackgroundColor": "#FFF2BE",
+                        "terminalFont": "IBM Plex Mono, 12",
```
Comment (mikekazakov) [inline Source/NimbleCommander/NimbleCommander/Resources/Config.json:940]: That doesn't seem like something macOS ships by default?

Comment (vladstudio) [reply Source/NimbleCommander/NimbleCommander/Resources/Config.json:940]: Ooh, true, missed that. I'll revert to Menlo.

Comment (mikekazakov) [reply Source/NimbleCommander/NimbleCommander/Resources/Config.json:940]: IIRC there are other monospaced fonts in macOS to try.

Comment (vladstudio) [reply Source/NimbleCommander/NimbleCommander/Resources/Config.json]: fixed: https://github.com/mikekazakov/nimble-commander/pull/122/commits/58bf3bf8b2903db659b14669f968d6291d97f839

**Comment by mikekazakov** [comment]
LGTM, thanks for adding the themes! 🥳
I realised that NC doesn't handle new built-in themes well if there were any overrides in the local config - the new themes might not show up, I will fix that soon.

**Pull request merged**